src/Tribe/Editor/REST/Compatibility.php

```diff
@@ -16,13 +16,19 @@ class Tribe__Tickets__Editor__REST__Compatibility {
 	 * @return boolean
 	 */
 	public function hook() {
+		// @todo: this does not appear to do _anything_ ?!?
```

Comment:
Should this be removed then? Have you tested without it?

Reply:
I have not - I was concerned about the reach into other plugins. If I have time before dev cutoff I will circle back and do so?

However, the implied purpose is to prevent those hooks being created during a REST request - which it appears they are not - as the new one works just fine.